src/SpendPermissionManager.sol

```diff
@@ -468,29 +471,29 @@ contract SpendPermissionManager is EIP712 {
         _transferFrom(spendPermission.token, spendPermission.account, spendPermission.spender, value);
     }
 
-    /// @notice Check if a spend permission is revoked.
+    /// @notice Get if a spend permission is approved.
     ///
     /// @param spendPermission Details of the spend permission.
     ///
-    /// @return revoked True if spend permission is revoked.
-    function isRevoked(SpendPermission memory spendPermission) public view returns (bool) {
-        return _isRevoked[getHash(spendPermission)];
+    /// @return approved True if spend permission is approved.
+    function isApproved(SpendPermission memory spendPermission) public view returns (bool) {
+        return _isApproved[getHash(spendPermission)];
```

Comment:
diff looks messed up here, but just swapped order of `isApproved` ahead of `isRevoked` for parity with non-view functions